ksql-engine/src/test/java/io/confluent/ksql/integration/StreamsSelectAndProjectIntTest.java

```diff
@@ -55,97 +61,191 @@ public void after() throws Exception {
+  }
+
+  private void testTimestampColumnSelection(String stream1Name,
+                                            String stream2Name,
+                                            String inputStreamName,
+                                            DataSource.DataSourceSerDe dataSourceSerDe,
+                                            Map<String, RecordMetadata> recordMetadataMap) throws Exception {
     final String query1String =
-            String.format("CREATE STREAM %s WITH (timestamp='RTIME') AS SELECT ROWKEY AS RKEY, "
-                            + "ROWTIME+10000 AS "
-                            + "RTIME, ROWTIME+100 AS RT100, ORDERID, ITEMID "
-                            + "FROM %s WHERE ORDERUNITS > 20 AND ITEMID = 'ITEM_8'; "
-                            + "CREATE STREAM %s AS SELECT ROWKEY AS NEWRKEY, "
-                            + "ROWTIME AS NEWRTIME, RKEY, RTIME, RT100, ORDERID, ITEMID "
-                            + "FROM %s ;", stream1Name,
-                    "ORDERS", stream2Name, stream1Name);
+        String.format("CREATE STREAM %s WITH (timestamp='RTIME') AS SELECT ROWKEY AS RKEY, "
+                      + "ROWTIME+10000 AS "
+                      + "RTIME, ROWTIME+100 AS RT100, ORDERID, ITEMID "
+                      + "FROM %s WHERE ORDERUNITS > 20 AND ITEMID = 'ITEM_8'; "
+                      + "CREATE STREAM %s AS SELECT ROWKEY AS NEWRKEY, "
+                      + "ROWTIME AS NEWRTIME, RKEY, RTIME, RT100, ORDERID, ITEMID "
+                      + "FROM %s ;", stream1Name,
+                      inputStreamName, stream2Name, stream1Name);
 
 
     ksqlContext.sql(query1String);
 
     Map<String, GenericRow> expectedResults = new HashMap<>();
-    expectedResults.put("8", new GenericRow(Arrays.asList(null, null, "8", recordMetadataMap.get("8")
-                    .timestamp() + 10000, "8", recordMetadataMap.get("8").timestamp() + 10000,
-            recordMetadataMap.get("8").timestamp
-                    () + 100, "ORDER_6", "ITEM_8")));
+    expectedResults.put("8", new GenericRow(Arrays.asList(null, null, "8", recordMetadataMap
+                                                                               .get("8")
+                                                                               .timestamp() + 10000, "8", recordMetadataMap.get("8").timestamp() + 10000,
+                                                          recordMetadataMap.get("8").timestamp
+                                                              () + 100, "ORDER_6", "ITEM_8")));
 
     Schema resultSchema = ksqlContext.getMetaStore().getSource(stream2Name).getSchema();
 
-    Map<String, GenericRow> results2 = testHarness.consumeData(stream2Name, resultSchema , expectedResults.size(), new StringDeserializer(), IntegrationTestHarness.RESULTS_POLL_MAX_TIME_MS);
+    Map<String, GenericRow> results2 = testHarness.consumeData(stream2Name, resultSchema ,
```

Comment:
if we are splitting params over multiple lines can we have 1 param per line and make them all line up? makes it much easier to read. Similarly elsewhere